server/src/storage/mod.rs

```diff
@@ -254,6 +262,39 @@ impl StorageEngine {
             .await
     }
 
+    /// Collect and return the logs of used uncompressed/compressed memory, used disk space, and ingested
+    /// data points over time. The logs are returned in tuples with the format (log_name, (timestamps, values)).
+    pub async fn collect_logs(&mut self) -> Vec<(String, (TimestampArray, UInt32Array))> {
+        vec![
+            (
+                "used_uncompressed_memory".to_owned(),
```

Comment:
Maybe convert `MetricType` to `String` (or more efficiently an int) in `remote.rs` as converting it to `String` in `collect_metrics()` leaves the return type of `collect_metrics()` as `Vec<(String, (TimestampArray, UInt32Array))>` instead of `Vec<(MetricType, (TimestampArray, UInt32Array))>` which would prevent returning an unsupported `MetricType`.

Reply:
Fixed.